yarn.lock

```diff
@@ -6099,6 +6099,7 @@ fs-extra@^8.1.0:
     jsonfile "^4.0.0"
     universalify "^0.1.0"
 
+<<<<<<< HEAD
```

Comment:
Can you fix these merge conflict artefacts?

Reply:
I think i have fixed it now